code/modules/spells/spell_types/lesserlichdom.dm

```diff
@@ -0,0 +1,151 @@
+		marked_item.desc += "\nA terrible aura surrounds this item, its very existence is offensive to life itself..."
+		marked_item.add_atom_colour("#187918", ADMIN_COLOUR_PRIORITY)
+
+		new /obj/item/lesserphylactery(marked_item, M.mind)
+
+		to_chat(M, "<span class='userdanger'>With a hideous feeling of emptiness you watch in horrified fascination as skin sloughs off bone! Blood boils, nerves disintegrate, eyes boil in their sockets! As your organs crumble to dust in your fleshless chest you come to terms with your choice. You're a lesser lich!</span>")
+		M.mind.hasSoul = FALSE
+		M.mind.hellbound = TRUE // No revival other than lichdom revival
+		M.set_species(/datum/species/skeleton)
+		// no robes spawn for a lesser spell
+		// you only get one phylactery.
+		M.mind.RemoveSpell(src)
+
+
+/obj/item/lesserphylactery
+	name = "lesser phylactery"
+	desc = "Stores souls. Revives lesser liches. Also repels mosquitos. Can only be used to revive a lich twice."
+	icon = 'icons/obj/projectiles.dmi'
+	icon_state = "bluespace"
+	color = "#187918"
+	light_color = "#187918"
+	light_system = MOVABLE_LIGHT
+	light_range = 3
+	var/lon_range = 3
+	var/resurrections = 0
+	var/datum/mind/mind
+	var/respawn_time = 3600  //Double the time of a regular phylactery
+
+	var/static/active_phylacteries = 0
+
+/obj/item/lesserphylactery/Initialize(mapload, datum/mind/newmind)
+	. = ..()
+	mind = newmind
+	name = "lesser phylactery of [mind.name]"
+
+	active_phylacteries++
+	GLOB.poi_list |= src
+	START_PROCESSING(SSobj, src)
+	if(initial(SSticker.mode.round_ends_with_antag_death))
+		SSticker.mode.round_ends_with_antag_death = FALSE
+
+/obj/item/lesserphylactery/Destroy(force=FALSE)
+	STOP_PROCESSING(SSobj, src)
+	active_phylacteries--
+	GLOB.poi_list -= src
+	if(!active_phylacteries)
+		SSticker.mode.round_ends_with_antag_death = initial(SSticker.mode.round_ends_with_antag_death)
+	. = ..()
+
+/obj/item/lesserphylactery/process()
+	if(QDELETED(mind))
+		qdel(src)
+		return
+
+	if(!mind.current || (mind.current && mind.current.stat == DEAD))
+		addtimer(CALLBACK(src, .proc/rise), respawn_time, TIMER_UNIQUE)
+
+/obj/item/lesserphylactery/proc/rise()
+	if(mind.current && mind.current.stat != DEAD)
+		return "[mind] already has a living body: [mind.current]"
+
+	var/turf/item_turf = get_turf(src)
+	if(!item_turf)
+		return "[src] is not at a turf? NULLSPACE!?"
+
+	var/mob/old_body = mind.current
+	var/mob/living/carbon/human/lich = new(item_turf)
+	// no robes spawn for lesser spell
+
+	lich.real_name = mind.name
+	mind.transfer_to(lich)
+	mind.grab_ghost(force=TRUE)
+	lich.hardset_dna(null,null,lich.real_name,null, new /datum/species/skeleton,null)
+	to_chat(lich, "<span class='warning'>Your bones clatter and shudder as you are pulled back into this world!</span>")
+	var/turf/body_turf = get_turf(old_body)
+	lich.Paralyze(400 + 200*resurrections) // paralyzed for longer due to lesser spell
+	resurrections++
+	if(old_body?.loc)
```

Comment:
early return this

Reply:
I have no idea what you mean, sorry

Reply:
if(!x)
   return

Reply:
for 	
if(old_body?.loc)
?
(this is pretty much just a copy of the original lich spell code)

Reply:
remove the nesting.
Put before the rest of the code
```
if(!old_body?.loc)
    return
```